### ISSUE TITLE
searches on twitter accounts

### ISSUE THREAD

**Issue by ghentsch** [linvi/tweetinvi#809]
Searches on accounts using from:<account> don't give the results i would expect. For example, in tweetinvi, the query is:

https://api.twitter.com/1.1/search/tweets.json?q=from%253Abannisterlake&result_type=recent&count=25&tweet_mode=extended

giving me 2 tweets whereas my handcoded query (not using tweetinvi) gives me 6:

https://api.twitter.com/1.1/search/tweets.json?q=-RT%20from%3Abannisterlake&result_type=recent&tweet_mode=extended&count=25

Could it be related to the extra escape in the tweetinvi query? %253A

**Comment by linvi** [comment]
Hi,

This is a bug that was reported after 4.0.
The bug has already been fixed by https://github.com/linvi/tweetinvi/pull/805.

The bug fix will be available in 5.0.

ETA: 1/2 weeks.

**Comment by linvi** [comment]
As a temporary fix you can do the following:

``` c#
TweetinviEvents.QueryBeforeExecute += (sender, eventArgs) =>
{
    var unescapedUrl = Uri.UnescapeDataString(eventArgs.TwitterQuery.QueryURL);
    eventArgs.TwitterQuery.QueryURL = unescapedUrl;
};
```
